### PR TITLE
【雪鱼】烧鹅-https://youtu.be/NIY6aPsSe3s

### PR DESCRIPTION
**Pull request by vincenthychow**
#553

### CODE REVIEW

**Comment by vincenthychow** [comment]
bot, please format

**Comment by immoonancient** [comment]
bot, please format

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 1]
```diff
@@ -0,0 +1,662 @@
...
+
+3
+00:00:08,960 --> 00:00:11,960
+# 还有旁边的 一条条的叉烧
+Next to them, strings of barbecue pork.
+
+4
+00:00:11,960 --> 00:00:14,720
+# 它们不仅颜色好看 外皮红亮
+Not only are they eye appealing, shiny red,
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
Not only are they visually appealing, shiny red,
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 2]
```diff
@@ -0,0 +1,662 @@
...
+
+5
+00:00:14,720 --> 00:00:16,880
+# 而且还流着汤汁
+but also the juice is dripping down.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
but also dripping juice
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 3]
```diff
@@ -0,0 +1,662 @@
...
+
+6
+00:00:17,020 --> 00:00:19,020
+# 太诱人了！
+It’s really seducing!
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
It’s really tempting!
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 4]
```diff
@@ -0,0 +1,662 @@
...
+
+7
+00:00:24,800 --> 00:00:28,320
+# 我们现在在大头华烧鹅店
+We are now in “Da Tou Hua Roast Geese Restaurant”
+
+8
+00:00:28,640 --> 00:00:31,760
+# 这不是传说当中的菜单吗
+Isn’t it the menu in the legend?
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
Legend has it that this is the menu
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 5]
```diff
@@ -0,0 +1,662 @@
...
+
+12
+00:00:38,560 --> 00:00:41,160
+# 也分大中小份
+And there are also large, medium and small portion
+
+13
+00:00:41,160 --> 00:00:43,760
+# 还有鹅杂三拼
+There’s also goose trio.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
There’s also goose giblets trio.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 6]
```diff
@@ -0,0 +1,662 @@
...
+
+14
+00:00:43,760 --> 00:00:47,480
+# 真的 鹅的全身都是宝 一样都跑不了
+The geese are really stuffed with treasure, everything.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
The geese are really stuffed with treasure, everything can be edible
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 7]
```diff
@@ -0,0 +1,662 @@
...
+
+19
+00:00:56,040 --> 00:00:57,040
+# 哇 还有鹅
+Wa! There are geese too.
+
+20
+00:00:57,640 --> 00:01:01,280
+# 这个是等一下 要去被风干的鹅
+These geese are to be dried.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
These geese are to be air dried.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 8]
```diff
@@ -0,0 +1,662 @@
...
+
+25
+00:01:12,640 --> 00:01:14,920
+# 我们现在（下来看一下）
+We now (come down and have a look)
+
+26
+00:01:15,120 --> 00:01:16,680
+# 大瓦缸
+It’s simple, this is a big porcelain tank.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
A big clay oven.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 9]
```diff
@@ -0,0 +1,662 @@
...
+
+30
+00:01:25,200 --> 00:01:28,240
+# 处理好的鹅 放在炉子里烤干些水分
+The processed geese are parched in the oven.
+
+31
+00:01:28,240 --> 00:01:30,920
+# 在鹅的皮上 洒脆皮卤水
+On the skin of the geese, we spray some crispy master stock.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
On the skin of the geese, we spray some crispyfy-ing master stock.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 10]
```diff
@@ -0,0 +1,662 @@
...
+
+33
+00:01:34,480 --> 00:01:37,080
+# 再进炉子里烤
+Before they’re roasted in the oven.
+
+34
+00:01:37,480 --> 00:01:39,240
+# 这个没有色素的
+It’s free of any colouring.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
It’s free of any food colouring.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 11]
```diff
@@ -0,0 +1,662 @@
...
+
+37
+00:01:43,040 --> 00:01:46,200
+# 八九斤
+Eight or nine “jin” (1 “jin” = 500 grams)
+
+38
+00:01:47,240 --> 00:01:50,280
+# 每一只鹅都气鼓鼓的
+The geese look flattened.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
The geese look puffy.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 12]
```diff
@@ -0,0 +1,662 @@
...
+
+57
+00:03:25,560 --> 00:03:28,080
+# 出去外面 温度都很高
+The temperature is really high outside.
+
+58
+00:03:28,080 --> 00:03:30,600
+# 你却把这个鹅皮搞干了
+But you dry the skin of the geese.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
By the time the skin of the geese is dried,
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 13]
```diff
@@ -0,0 +1,662 @@
...
+
+59
+00:03:30,600 --> 00:03:33,120
+# 里面的肉就变了
+The meat may be rotten.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
the meat inside has deteriorated.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 14]
```diff
@@ -0,0 +1,662 @@
...
+
+69
+00:04:02,560 --> 00:04:05,840
+# 一般做烧鹅 没有这个流程
+Other roast geese do not go through this step.
+
+70
+00:04:05,840 --> 00:04:09,200
+# 没有这个流程的话 皮可能就没有那么脆
+The skin won’t be that crispy without this stel.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
The skin won’t be that crispy without this step.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 15]
```diff
@@ -0,0 +1,662 @@
...
+
+75
+00:04:41,560 --> 00:04:43,880
+# 一个大头华烧鹅
+One is “Da Tou Hua oast geese”.
+
+76
+00:04:43,880 --> 00:04:46,880
+# 一个腊肉蒸鲮鱼干
+The other one is “cured pork steamed with dried catfish”
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
The other one is “cured pork steamed with dried carp”
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 16]
```diff
@@ -0,0 +1,662 @@
...
+
+77
+00:05:04,080 --> 00:05:09,640
+# 现在在上面 应该有北风 有太阳
+It should be up there. There’s northern wind, and sun.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
It should be up there. There’s north wind, and sun.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 17]
```diff
@@ -0,0 +1,662 @@
...
+
+80
+00:05:38,200 --> 00:05:40,560
+# 500多条鱼
+More than half a thousand fish
+
+81
+00:05:42,160 --> 00:05:44,840
+# 叔叔 这个鱼得晒多久
+Uncle, how long does it take to dry the fish?
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
Uncle, how long does it take to sun dry the fish?
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 18]
```diff
@@ -0,0 +1,662 @@
...
+
+93
+00:07:44,440 --> 00:07:47,280
+# 一个小时 可以出炉
+They will be ready in an hour.
+
+94
+00:07:48,180 --> 00:07:50,760
+# 用瓦缸做烧鹅的方法 得几百年了
+The method of using a porcelain tank is centuries old.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
The method of using a clay oven is centuries old.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 19]
```diff
@@ -0,0 +1,662 @@
...
+
+96
+00:07:57,120 --> 00:08:02,200
+# 因为这个炉子 有一些优点
+As the oven has its own strengths.
+
+97
+00:08:02,200 --> 00:08:04,040
+# 保温
+It keeps the moisture.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
It retains heat very well.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 20]
```diff
@@ -0,0 +1,662 @@
...
+
+100
+00:08:11,680 --> 00:08:15,160
+# 含有传统 古老的味道
+The difference is the traditional, nostalgic taste.
+
+101
+00:08:15,800 --> 00:08:17,360
+# 鹅已经就位了
+The geese are not ready.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
The geese are now ready.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 21]
```diff
@@ -0,0 +1,662 @@
...
+
+104
+00:08:23,120 --> 00:08:26,600
+# 叔叔 每天的工作就是提着鹅到处跑
+Uncle! So your job is taking the geese here and there every day?
+
+105
+00:08:56,000 --> 00:08:59,200
+# 这个最厉害 这都是鹅肉原汁
+This is the most attractive one, the juice from the goose.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
This is the most impressive, the juice from the goose.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 22]
```diff
@@ -0,0 +1,662 @@
...
+
+106
+00:09:06,880 --> 00:09:09,200
+# 刚刚出炉的好吃
+The tastes the best when it just comes out from the oven.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
It tastes the best when it just comes out from the oven.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 23]
```diff
@@ -0,0 +1,662 @@
...
+
+116
+00:10:42,120 --> 00:10:44,920
+# 真的是 皮脆 肉嫩 多汁
+It’s really, crispy outside and juicy inside.
+
+117
+00:10:44,920 --> 00:10:46,400
+# 还有呢 如果觉得腻
+And, if you feel greesy,
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
And, if you feel greasy,
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 24]
```diff
@@ -0,0 +1,662 @@
...
+
+120
+00:10:55,440 --> 00:10:57,480
+# 这个蜜汁叉烧
+This honey barbecue pork.
+
+121
+00:10:57,480 --> 00:10:59,880
+# 真的会有迷一般的魅力
+is really seducing.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
It's fascinating.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 25]
```diff
@@ -0,0 +1,662 @@
...
+
+124
+00:11:04,560 --> 00:11:06,240
+# 味道呢 会比较淡一些
+The tastes is a bit more mild.
+
+125
+00:11:06,240 --> 00:11:09,160
+# 鹅肠 就是更加爽脆
+And the goose intestines are exceptionally crunchy.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
And the goose intestines are more crunchy.
```

File: subtitles/xue-yu/20200104-roast-geese.srt [hunk 26]
```diff
@@ -0,0 +1,662 @@
...
+
+128
+00:11:16,080 --> 00:11:18,040
+# 叉烧是送的
+The barbecue pork is complementary.
+
+129
+00:11:18,040 --> 00:11:22,160
+# 好一桌的味觉体验
+What a table of experience of taste.
```
Comment (NoraZheng) [inline subtitles/xue-yu/20200104-roast-geese.srt]:
```suggestion
What a gastronomic experience.
```

**Pull request merged**